modules/xmpp/SignalingLayerImpl.js

```diff
@@ -234,6 +244,14 @@ export default class SignalingLayerImpl extends SignalingLayer {
             const endpointId = Strophe.getResourceFromJid(jid);
 
             delete this._remoteSourceState[endpointId];
+
+            if (FeatureFlags.isSourceNameSignalingEnabled()) {
+                for (const [key, value] of this.ssrcOwners.entries()) {
+                    if (value === endpointId) {
+                        delete this._sourceNames[key];
```

Comment:
great, can you put a comment here with a note that this may stop working if ssrcOwners are cleaned up before this code executes?

Reply:
Are we deleting the entry from `this._sourceNames` map when a `source-remove` is received from Jicofo ?

Reply:
oh that's a good point, but we weren't doing that for the SSRC owners either

Reply:
Yes, we need to clean up SSRC owners as well. I didn't notice this until now.

Reply:
I was thinking the same thing , needing to delete it for source-remove , but did not think that was within scope of this PR. Should that be a follow up PR or should we include that here?

Reply:
I think it's ok to do it in a follow up PR.